README.rst

```diff
@@ -47,24 +47,32 @@ Here's an example
 .. code-block:: python
 
     # mycode.py
+    import procrastinate
 
     # Make an app in your code
-    app = procrastinate.App(connector=procrastinate.AiopgConnector())
+    app = procrastinate.App(connector=procrastinate.AiopgConnector(host="localhost", user="postgres", password="password"))
```

Comment:
In my opinion the goal of the code snippets in the README is to show what Procrastinate is and the various constructs it provides, but they're not necessarily codes you can copy & paste and execute. We have the Quickstart for that. So I'd rather keep the README snippets simple and to the point, avoiding things like passing database parameters to the `AiopgConnector` constructor.